tools/gfx/vulkan/vk-device.cpp

```diff
@@ -2154,6 +2257,20 @@ Result DeviceImpl::createBufferView(
                 info.buffer = resourceImpl->m_buffer.m_buffer;
                 info.offset = offset;
                 info.range = size;
+                VkBufferUsageFlags2CreateInfoKHR bufferViewUsage;
+                bufferViewUsage = {};
+                bufferViewUsage.sType = VK_STRUCTURE_TYPE_BUFFER_USAGE_FLAGS_2_CREATE_INFO_KHR;
+                info.pNext = &bufferViewUsage;
+                if (desc.type == IResourceView::Type::UnorderedAccess)
+                {
+                    bufferViewUsage.usage = VK_BUFFER_USAGE_2_STORAGE_TEXEL_BUFFER_BIT_KHR;
+                }
+                else if (desc.type == IResourceView::Type::ShaderResource)
+                {
+                    bufferViewUsage.usage = VK_BUFFER_USAGE_2_UNIFORM_TEXEL_BUFFER_BIT_KHR;
+                }
+                else
+                    assert(!"unhandled");
```

Comment:
Please wrap the line with "{" and "}".
Depending on how "assert" macro is defined, this line could disappear and unwanted next line may become a part of "else".

Reply:
I had no idea assert is a macro sometimes, thank you for sharing~ I 100% should NOT be doing this with assert...